### ISSUE TITLE
JS version returns ERR_BUSY instead of ERR_TIRED if fatigue > 0

### ISSUE THREAD

**Issue by poma** [bonzaiferroni/Traveler#20]
What .js returns: [Traveler.js#L23](https://github.com/bonzaiferroni/Traveler/blob/master/Traveler.js#L23)

What .ts returns: [Traveler.ts#L32](https://github.com/bonzaiferroni/Traveler/blob/master/Traveler.ts#L32)

**Comment by bonzaiferroni** [comment]
Good catch! I haven't actually compiled the code myself for the last few pull requests, so that probably explains it. I've updated the js file manually.
